integrations/torchvision/README.md

```diff
@@ -0,0 +1,202 @@
+
+For full usage, run:
+```bash
+sparseml.image_classification --help
+```
+
+### SparseML Python API
+
+For additional flexibility, SparseML has a Python API that also enables you to add sparsification to a native PyTorch training loop.
+
+Just like the CLI, the Python API uses YAML-based recipes to encode the parameters of the sparsification process, allowing youto add SparseML with just a few lines of code.
+
+The `ScheduleModifierManager` class is responsible for parsing the YAML recipes and overriding the standard PyTorch model and optimizer objects, 
+encoding the logic of the sparsity algorithms from the recipe. Once you call `manager.modify`, you can then use the model and optimizer as usual, as SparseML abstracts away the complexity of the sparsification algorithms.
+
+The workflow looks like this:
+
+```python
+# typical model, optimizer, dataset definition
+model = Model()
+optimizer = Optimizer()
+train_data = TrainData()
+
+# parse recipe + edit model/optimizer with sparsity-logic
+from sparseml.pytorch.optim import ScheduledModifierManager
+manager = ScheduledModifierManager.from_yaml(PATH_TO_RECIPE)
+optimizer = manager.modify(model, optimizer, len(train_data))
+
+# PyTorch training loop, using the model/optimizer as usual
+
+# clean-up
+manager.finalize(model)
+```
+
+See the tutorials for full working examples of the Python API.
+
+## Quick Start: Sparse Transfer Learning
+
+### Overview
+
+Sparse Transfer is quite similiar to the typical transfer learing process used to train NLP models, where we fine-tune a pretrained checkpoint onto a smaller downstream dataset. With Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity while the training process occurs.
+
+In this example, we will fine-tune a 95% pruned version of ResNet-50 ([available in SparseZoo](https://sparsezoo.neuralmagic.com/models/cv%2Fclassification%2Fresnet_v1-50%2Fpytorch%2Fsparseml%2Fimagenet%2Fpruned95_quant-none)) onto ImageNette.
+
+### Kick off Training
+
+We can start Sparse Transfer Learning by passing a starting checkpoint and recipe to the training script. For Sparse Transfer, we will use a recipe that instructs SparseML to maintain sparsity during training and to quantize the model. The starting checkpoitn and transfer recipe are specified by the following SparseZoo stub:
+```
+zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none?recipe_type=transfer-classification
+```
+
+<details>
+   <summary>Click to see the recipe</summary>
+
+SparseML parses the `Modifers` in the recipe and updates the training loop with logic encoded therein.
+   
+The key `Modifiers` for sparse transfer learning are the following:
+- `ConstantPruningModifier` instructs SparseML to maintain the sparsity structure of the network during the fine-tuning process
```

Comment:
In other places we say "ConstantPruningModifier tells SparseML to pin weights at 0 over all epochs, maintaining the sparsity structure of the network". Are they the same thing?